### ISSUE TITLE
Compat issues on julia 1.11

### ISSUE THREAD

**Issue by activexray** [torrance/Casacore.jl#12]
```
   Resolving package versions...
ERROR: Unsatisfiable requirements detected for package casacorecxx_jll [689e2960]:
 casacorecxx_jll [689e2960] log:
 ├─possible versions are: 0.1.0 - 0.3.0 or uninstalled
 ├─restricted by julia compatibility requirements to versions: uninstalled
```

I'm going to attempt to give everything a little update from casacore_jll up through the Cxx wrap stuff.

**Comment by activexray** [comment]
`casacore` only moved one minor version that fixed compat with numpy 2, so not important to us so I won't bump it

**Comment by activexray** [comment]
Ah however I realize now in the casacore JLL, I'm linking against OpenBLAS instead of libblastrampoline. This could result in some pretty significant speedups with Apple Accelerate/MKL/etc. I'll go ahead and update that and work my way down. However - LBT needs at least Julia 1.9, and now Julia 1.10 is the LTS release, so I think I'll bump all the compat to that

**Comment by activexray** [comment]
Working on casacore_jll updates here: https://github.com/JuliaPackaging/Yggdrasil/pull/9964

**Comment by haavee** [comment]
Hi there!
I was wondering what the status of this is, or if I could build your package locally (I don't think I can).
On my Mac/M1 the homebrew julia is either 1.10.4 or 1.11 and trying to (manually) build it using BinaryBuilder that fails spectacularly because that one says it can't work with anything beyond 1.7 ...

**Comment by barrettp** [comment]
Yes, it would be nice if this package worked with Julia 1.11. Presently, it will not install because of a version issue with casacorecxx_jll.

**Comment by activexray** [comment]
Now that casacore_jll has been updated, I think I'll to finish this up.
